simulator-core/src/test/java/com/evbox/everon/ocpp/mock/StationSimulatorSetUp.java

```diff
@@ -0,0 +1,62 @@
+package com.evbox.everon.ocpp.mock;
+
+import com.evbox.everon.ocpp.mock.constants.StationConstants;
+import com.evbox.everon.ocpp.mock.factory.SimulatorConfigCreator;
+import com.evbox.everon.ocpp.mock.ocpp.OcppMockServer;
+import com.evbox.everon.ocpp.mock.ocpp.OcppServerClient;
+import com.evbox.everon.ocpp.mock.ocpp.exchange.BootNotification;
+import com.evbox.everon.ocpp.simulator.StationSimulatorRunner;
+import com.evbox.everon.ocpp.simulator.configuration.SimulatorConfiguration;
+import com.evbox.everon.ocpp.simulator.station.StationMessage;
+import com.evbox.everon.ocpp.simulator.station.actions.UserMessage;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+
+import java.util.concurrent.TimeUnit;
+
+import static com.evbox.everon.ocpp.mock.constants.StationConstants.*;
+import static com.evbox.everon.ocpp.mock.expect.ExpectedCount.any;
+
+public class StationSimulatorSetUp  {
+
+    private static final int ASSERT_TIMEOUT_IN_SECONDS = 10;
+
+    protected OcppServerClient ocppServerClient = new OcppServerClient();
+
+    protected OcppMockServer ocppMockServer = OcppMockServer.builder()
+            .hostname(StationConstants.HOST)
+            .port(StationConstants.PORT)
+            .path(StationConstants.PATH)
+            .ocppServerClient(ocppServerClient)
+            .build();
+
+    protected StationSimulatorRunner stationSimulatorRunner;
+
+    @BeforeEach
+    void setUp() {
+        Awaitility.setDefaultTimeout(ASSERT_TIMEOUT_IN_SECONDS, TimeUnit.SECONDS);
+
+        ocppMockServer.start();
+
+        SimulatorConfiguration.StationConfiguration stationConfiguration = SimulatorConfigCreator.createStationConfiguration(STATION_ID, DEFAULT_EVSE_COUNT, DEFAULT_EVSE_CONNECTORS);
+        SimulatorConfiguration simulatorConfiguration = SimulatorConfigCreator.createSimulatorConfiguration(stationConfiguration);
+
+        stationSimulatorRunner = new StationSimulatorRunner(StationConstants.OCPP_SERVER_URL, simulatorConfiguration);
+
+        // Station always needs boot notification message on startup
+        ocppMockServer
+                .when(BootNotification.request(), any())
```

Comment:
Please change that to once(), as station should send it only once

Reply:
done

Reply:
okay, I see. Then atLeastOnce will do the job

Reply:
done